src/tpm.rs

```diff
@@ -24,3 +25,36 @@ pub(crate) fn get_tpm2_ctx() -> Result<tss_esapi::Context> {
     let tcti = Tcti::from_str(tcti_path)?;
     unsafe { Context::new(tcti) }.map_err(|e| e.into())
 }
+
+/*
+ * Input: Connection context, asymmetric algo (optional)
+ * Return: (Key handle, public cert)
+ * Example call:
+ * let (key, cert) = tpm::create_ek(context, Some(AsymmetricAlgorithm::Rsa))
+ */
+pub(crate) fn create_ek(
+    context: &mut Context,
+    alg: Option<AsymmetricAlgorithm>,
+) -> Result<(KeyHandle, Vec<u8>)> {
+    let alg = match alg {
+        Some(a) => a,
+        None => {
+            // TODO: What other values could be in keylime.conf?
```

Comment:
I think the only other value we should consider correct is `ecc`.
I'd say that the `_ =>` should return an error instead.